frontend/public/components/factory/ListPage/ListPageCreate.tsx

```diff
@@ -111,8 +113,8 @@ const ListPageCreate: React.FC<ListPageCreateProps> = ({
       : `/k8s/cluster/${k8sModel.plural}/~new`;
     if (k8sModel.crd) {
       to = usedNamespace
-        ? `/k8s/ns/${usedNamespace || 'default'}/${groupVersionKind}/~new`
-        : `/k8s/cluster/${groupVersionKind}/~new`;
+        ? `/k8s/ns/${usedNamespace || 'default'}/${kind}/~new`
```

Comment:
We want to make sure we're using the "reference" string here which is `~` delimited for resources in an API group.